api/swagger-spec/storage.k8s.io_v1beta1.json

```diff
@@ -675,7 +675,7 @@
      },
      "metadata": {
       "$ref": "v1.ObjectMeta",
-      "description": "Standard object's metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata"
```

Comment:
I cant find the script and I see our api reference docs on k8s.io are referring to HEAD instead of a release version: https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/api-reference/extensions/v1beta1/definitions.html#L246.
So looks like we broke it. Dont think that was intentional.